packages/rum-core/src/domain/assembly.ts

```diff
@@ -96,9 +99,10 @@ export function startRumAssembly(
           date: timeStampNow(),
           service: configuration.service,
           session: {
-            type: syntheticsContext ? SessionType.SYNTHETICS : SessionType.USER,
+            type: syntheticsContext ? SessionType.SYNTHETICS : ciContext ? SessionType.CITEST : SessionType.USER,
```

Comment:
You need to submit this new type in [rum-event-format](https://github.com/DataDog/rum-events-format).

Reply:
will do, thanks!

Reply:
PR in rum-event-format: https://github.com/DataDog/rum-events-format/pull/49